.github/workflows/ci.yml

```diff
@@ -528,7 +528,7 @@ jobs:
       - name: Downgrade ESLint to 7.5.0
         run: yarn up eslint@7.5.0
       - name: Run babel/eslint tests
-        run: node ./node_modules/.bin/jest eslint
+        run: node ./node_modules/.bin/jest eslint/babel-eslint-parser/ eslint/babel-eslint-plugin/ babel-eslint-tests/
```

Comment:
Why do we need this change?

Reply:
https://github.com/babel/babel/actions/runs/8980473754/job/24664241688
Our internal plugins no longer support the old eslint.

Reply:
`eslint-plugin-development` appears to be public, despite only having a few hundred weekly downloads, should we keep compatibility?

Reply:
Whops, I missed this. Yes, we should keep compatibility.